.gitignore

```diff
@@ -3,5 +3,6 @@
 *.mo
 **/data/*.json
 !localhost/data/[client|server].json
+localhost/server/__pycache__/
```

Comment:
@thekaveman Not sure if I should make this directory more flexible with asterisks.